.github/workflows/test.yaml

```diff
@@ -5,7 +5,7 @@ on:
 
 jobs:
   unit-tests:
-    uses: canonical/operator-workflows/.github/workflows/test.yaml@main
```

Comment:
Is this intentional?

Reply:
Kind of... as discourse-gatekeeper will be updated, it is necessary for NetBox to work for now until all is merged.

Reply:
Understood